### ISSUE TITLE
Too many files open

### ISSUE THREAD

**Issue by sanegaming** [pterodactyl/panel#4657]
### Current Behavior

On multiple nodes of mine throughout the day I am getting the "Too many files open" and the wings node no longer being responsive errors.

### Expected Behavior

I would it expect to not let this happen and do some cleanup if needed?

### Steps to Reproduce

On one of my nodes, I have 5 servers, (2 Vanilla Minecraft, 1 Modded MC, 1 Discord bot, and 1 fivem server)

The node has 72GB of Ram assigned, running as a VM within a Proxmox cluster.

While they are all running and being used throughout the day the error happens.

### Panel Version

1.11.2

### Wings Version

1.11.0

### Games and/or Eggs Affected

Wings

### Error Logs

```bash
open /var/lib/pterodactyl/volumes/408cabdf-5190-490c-9eda-04e893c2859b: too many open files
Jan 17 14:23:15 Wings3 wings[2019]: server/filesystem: directorysize: failed to walk directory
Jan 17 14:23:15 Wings3 wings[2019]: github.com/pterodactyl/wings/server/filesystem.(*Filesystem).DirectorySize
Jan 17 14:23:15 Wings3 wings[2019]:         github.com/pterodactyl/wings/server/filesystem/disk_space.go:194
Jan 17 14:23:15 Wings3 wings[2019]: github.com/pterodactyl/wings/server/filesystem.(*Filesystem).updateCachedDiskUsage
Jan 17 14:23:15 Wings3 wings[2019]:         github.com/pterodactyl/wings/server/filesystem/disk_space.go:145
Jan 17 14:23:15 Wings3 wings[2019]: github.com/pterodactyl/wings/server/filesystem.(*Filesystem).DiskUsage.func1
Jan 17 14:23:15 Wings3 wings[2019]:         github.com/pterodactyl/wings/server/filesystem/disk_space.go:116
Jan 17 14:23:15 Wings3 wings[2019]: runtime.goexit
Jan 17 14:23:15 Wings3 wings[2019]:         runtime/asm_amd64.s:1571
Jan 17 14:23:15 Wings3 wings[2019]: 2023/01/17 14:23:15 http: Accept error: accept tcp [::]:10100: accept4: too many open files; retrying in 1s
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by hwalker928** [comment]
Fixed in wings develop: https://github.com/pterodactyl/wings/commit/e9b8b11fec6075694d5962a6d65245d121d191d9

**Comment by sanegaming** [comment]
> Fixed in wings develop: [pterodactyl/wings@e9b8b11](https://github.com/pterodactyl/wings/commit/e9b8b11fec6075694d5962a6d65245d121d191d9)

Awesome thanks, do you know when this will get released?

**Comment by FoksVHox** [comment]
> > Fixed in wings develop: [pterodactyl/wings@e9b8b11](https://github.com/pterodactyl/wings/commit/e9b8b11fec6075694d5962a6d65245d121d191d9)
> 
> Awesome thanks, do you know when this will get released?

@sanegaming It'll be released in the next version, which will properly be either a bug fix version, a minor version bump or v2 major version.
